enhancements/10x/firefix.sp

```diff
@@ -0,0 +1,53 @@
+#include <sourcemod>
+#include <sdktools>
+#include <sdkhooks>
+#include <rxgcommon>
+#include <tf2_stocks>
+
+#pragma semicolon 1
+#pragma newdecls required
+
+public Plugin myinfo = {
+	name = "Fire Fix",
+	author = "Dooby Skoo",
+	description = "Ignites players who are hit with specified weapons.",
+	version = "1.0.0",
+	url = "www.reflex-gamers.com"
+};
+
+int validWeapons[] =  { 264, 171, 307, 1071, 349, 648, 666 };
+
+//-----------------------------------------------------------------------------
+public void OnPluginStart() {
+	for( int client = 1; client <= MaxClients; client++ ) {
+		if (!IsValidClient(client)) { continue; }
+		SDKHook(client, SDKHook_OnTakeDamage, OnTakeDamage);
+	}
+}
+
+//-----------------------------------------------------------------------------
+public void OnClientPutInServer( int client ) {
+	SDKHook(client, SDKHook_OnTakeDamage, OnTakeDamage);
+}
+
+//-----------------------------------------------------------------------------
+public Action OnTakeDamage( int victim, int &attacker, int &inflictor,
+							float &damage, int &damagetype, int &weapon,
+							float damageForce[3], float damagePosition[3],
+							int damagecustom) {

```

Comment:
Awkward spacing on function arguments here; align with `int victim`

Reply:
This is likely due to GitHub using 8 spaces for tabs.